compiler-core/src/metadata/tests.rs

```diff
@@ -1557,6 +1630,8 @@ fn type_with_inferred_variant() {
         line_numbers: LineNumbers::new(""),
         src_path: "some_path".into(),
         minimum_required_version: Version::new(0, 1, 0),
+        type_aliases: HashMap::new(),
+        documentation: Vec::new(),
```

Comment:
Please add some new tests when new functionality is added to the metadata encoding and decoding. This is the area of the codebases that causes the most bugs